src/scss/3_components/form/form.scss

```diff
@@ -0,0 +1,111 @@
+/*
+ * @file
+ * Provides the generic styling for all types of form components.
+ */
+@import '../../init';
+
+fieldset,
+.fieldset {
+  border: 0;
+
+  &:focus {
+    outline: 0;
+    border: 0;
+  }
+}
+
+select,
+[type='text'],
+[type='tel'],
+[type='url'],
+[type='email'],
+[type='password'],
+[type='date'],
+[type='time'],
+[type='number'],
+//input:not([type='button']):not([type='submit']):not([type='radio']):not([type='checkbox']):not([type='search']),
+textarea {
+  box-sizing: border-box;
+  border: 1px solid #333;
+  border-radius: 0;
+  color: #333;
+  margin: .6rem 0 0;
+  padding: .6rem;
+
+  &:focus {
+    border-color: #2daff7;
+    box-shadow: 0 0 3px #2daff7;
+    color: $black;
+    outline: 0;
+  }
+
+  /* Highlight the form elements that caused a form submission error. */
+  &.error,
+  .error:focus {
+    border: 1px solid #c94b4b;
+    background-color: rgba(255, 182, 193, .5); // lightpink
+    box-shadow: none;
+    color: #000;
+  }
+
+  @include breakpoint($phab, $no-query: true) {
+    margin: .9rem 0 0;
+    padding: .6rem;
+  }
+}
+
+// Based on https://css-tricks.com/styling-a-select-like-its-2019/
+.form-select {
+  display: block;
+  font-size: 16px;
+  font-family: sans-serif;
+  font-weight: normal;
+  color: $charcoal;
+  line-height: 1.3;
+  padding: .6em 1.4em .5em .8em;
+  box-sizing: border-box;
+  margin: rhythm(1) 0 0;
+  border: 1px solid $grey-dark;
+  border-radius: 0;
+  -moz-appearance: none;
```

Comment:
Prefixer should negate the need for these.

Reply:
indeed. As per last, this file is a work in progress so a lot to still sort on it, this was just a basic start to help me with bugs around the search form on health condition pages